### ISSUE TITLE
Verson 2: Switch from`semver` to `semantic_version` package.

### ISSUE THREAD

**Issue by pipermerriam** [pipermerriam/populus#308]
* Populus Version: 2.x.x (on the `feat/v2` branch)

### What is wrong?

Currently, in the `feat/v2` line of Populus, the `semver` package is used for semantic versioning.  This should switch to use the `semantic_version` package found here: https://pypi.python.org/pypi/semantic_version

**Comment by nemaniarjun** [comment]
Hey! I am trying to do this, but when I install from source and try to run `populus` it gives me an error regarding, `ethereum_utils` package. 

So I updated the dependencies in `setup.py` from `ethereum_utils` to `eth_utiles`, but this gives me an Import Error  ```ImportError: cannot import name 'force_bytes'```

**Comment by nemaniarjun** [comment]
Probably related to: https://github.com/ethereum/populus/issues/447

**Comment by nemaniarjun** [comment]
Never mind, fixed it temporarily using `pip3 install eth-utils==0.7.4`

**Comment by nemaniarjun** [comment]
This keeps giving me errors regarding `eth_utils`

**Comment by marbetschar** [comment]
right, https://github.com/ethereum/populus/issues/450 needs to be fixed before this issue can be resolved.
